vue.config.js

```diff
@@ -52,6 +60,57 @@ module.exports = {
         }
       }
     }
+    return {
+      optimization: {
+        runtimeChunk: 'single',
+        splitChunks: {
+          cacheGroups: {
+            vendor: {
+              test: /[\\/]node_modules[\\/](?!(@fortawesome|mavon-editor|v-calendar|vee-validate|vue-moment|vue-toasted))(.[a-zA-Z0-9.\-_]+)[\\/]/,
+              chunks: 'initial',
+              name: 'vendors',
+              enforce: true
+            },
+            fortawesome: {
+              test: /[\\/]node_modules[\\/]@fortawesome(.[a-zA-Z0-9.\-_]+)[\\/]/,
+              chunks: 'initial',
+              name: 'fortawesome',
+              enforce: true
+            },
+            mavon_editor: {
+              test: /[\\/]node_modules[\\/]mavon-editor(.[a-zA-Z0-9.\-_]+)[\\/]/,
+              chunks: 'initial',
+              name: 'mavon-editor',
+              enforce: true
+            },
+            v_calendar: {
+              test: /[\\/]node_modules[\\/]v-calendar(.[a-zA-Z0-9.\-_]+)[\\/]/,
+              chunks: 'initial',
+              name: 'v-calendar',
+              enforce: true
+            },
+            vee_validate: {
+              test: /[\\/]node_modules[\\/]vee-validate(.[a-zA-Z0-9.\-_]+)[\\/]/,
+              chunks: 'initial',
+              name: 'vee-validate',
+              enforce: true
+            },
+            vue_moment: {
+              test: /[\\/]node_modules[\\/]vue-moment(.[a-zA-Z0-9.\-_]+)[\\/]/,
+              chunks: 'initial',
+              name: 'vue-moment',
+              enforce: true
+            },
+            vue_toasted: {
+              test: /[\\/]node_modules[\\/]vue-toasted(.[a-zA-Z0-9.\-_]+)[\\/]/,
```

Comment:
this as well

Reply:
See https://github.com/function-future/function-frontend/pull/239#discussion_r371592813, with difference the result would be no `vue-toasted.js` chunk. @oliverrsebastian